rasa_sdk/grpc_errors.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from pydantic import BaseModel, Field
+
+from enum import Enum
+
+
+class ActionExecutionFailed(BaseModel):
```

Comment:
These errors are used to provide more context to the receiver into what actually happened.